apps/server/src/infra/etherpad-client/loggable/etherpad-server-error-exception.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { ErrorUtils } from '@src/core/error/utils';
+import { ErrorType } from '../interface';
+import { EtherpadServerError } from './etherpad-server-error-exception';
+
+describe('EtherpadServerErrorException', () => {
```

Comment:
I'm pretty sure we decided that loggable exceptions don't need to be tested.

Reply:
Remove tests.